qa/tasks/cephfs/mount.py

```diff
@@ -124,13 +124,15 @@ def create_destroy(self):
             'sudo', 'rm', '-f', os.path.join(self.mountpoint, filename)
         ])
 
-    def _run_python(self, pyscript):
-        return self.client_remote.run(args=[
-            'sudo', 'adjust-ulimits', 'daemon-helper', 'kill', 'python', '-c', pyscript
-        ], wait=False, stdin=run.PIPE, stdout=StringIO())
-
-    def run_python(self, pyscript):
-        p = self._run_python(pyscript)
+    def _run_python(self, pyscript, use_python3=False):
+        py_exe = 'python2' if use_python3 else 'python3'
```

Comment:
Default should be simply 'python', i.e. whatever the system default is. The parameter should be `use_python='python'` so the caller can specify `python3` or `python2` for testing. The CephVolumeClient code msut be backwards/forwards compat.

Reply:
Done. Changed the variable name, though, from `use_python` to `py_version`.